pcsx2/PluginManager.cpp

```diff
@@ -420,6 +422,7 @@ static const LegacyApi_ReqMethod s_MethMessReq_GS[] =
 	{	"GSsetExclusive",	(vMeth**)&GSsetExclusive,	(vMeth*)GS_setExclusive	},
 	{	"GSchangeSaveState",(vMeth**)&GSchangeSaveState,(vMeth*)GS_changeSaveState },
 	{	"GSgetTitleInfo2",	(vMeth**)&GSgetTitleInfo2,	(vMeth*)GS_getTitleInfo2 },
+	{	"GSgetImageSize",	(vMeth**)&GSgetImageSize,	(vMeth*)GS_getImageSize },
```

Comment:
It should be moved to `s_MethMessOpt_GS` instead. And a check must be done if the pointer is nullptr before using it.